### PR TITLE
Fix Issue #2441: seperated debug & release icons

### PR DESCRIPTION
**Pull request by danger-ahead**
#### Issue tracker   
#2441 

Fixes #
Added label in debug build icon

#### Release  
Addresses release/3.6
  
#### Test cases
- [x] Covered
  
#### Manual testing
- [x] Done  
  
If yes,  
- Device: Infinix Note 5
- OS: Android 10

#### Build tasks success  
Successfully running following tasks on local 
- [x] `./gradlew assembledebug`
- [x] `./gradlew spotlessCheck`

#### Related PR  
Related to PR #

#### Additional Info
image
image

### CODE REVIEW

**Review by VishnuSanal — APPROVED** [review]
Looks good to me :)

File: app/build.gradle [hunk 1]
```diff
@@ -3,6 +3,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.hiya.jacoco-android'
+apply plugin: 'com.akaita.android.easylauncher'
```
Comment (EmmanuelMess) [inline app/build.gradle:6]: Does this work with other launchers?

Comment (danger-ahead) [reply app/build.gradle:6]: Yes, I've tasted this on Samsung's One UI too.

Comment (danger-ahead) [reply app/build.gradle:6]: @EmmanuelMess For your reference:
image
image

**Comment by TranceLove** [comment]
Many thanks for your work!

For others which you may experience awkward icon backgrounds, it may be due to the incorrect icon background we choose.

On stock Android 11 Launcher3 and Lawnchair on my Fairphone 3 16.0(9.0) both variants give an opaque white background, while the release variant on Bliss Launcher on my Oneplus 2 running /e/ OS gives an opaque blue background instead.

Screenshot_20210524-102051_Bliss_Launcher

Icon background problem should be another issue (if it is). So I will vote +1 here.

File: app/build.gradle (hunk 1 above)
Comment (EmmanuelMess) [reply app/build.gradle:6]: Works on KISS too.

**Review by EmmanuelMess — APPROVED** [review]

**Pull request merged**